### PR TITLE
Add Symfony 4 compatibility

### PR DESCRIPTION
**Pull request by cebcardoso**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -38,8 +39,16 @@
     "require-dev": {
         "ext-mcrypt": "*",
         "mikey179/vfsStream": "~1.0",
-        "symfony/filesystem": "~2.3",
-        "symfony/process": "~2.3"
+        "symfony/filesystem": "^4.3",
+        "symfony/form": "^4.3",
+        "symfony/process": "^4.3",
```
Comment (smithandre) [inline composer.json]: Why do we need to include all these dependencies suddenly? Seems a bit strange also to have something like messenger or forms as dev requirements? Or is it just me missing something?

Comment (cebcardoso) [reply composer.json]: I included this updated branch in our update PR for MIA and it complained that this component didn't have these dependencies.

Comment (cebcardoso) [reply composer.json]: Should I remove them?

Comment (smithandre) [reply composer.json]: > Should I remove them?

I am not sure.

File: composer.json [hunk 2]
```diff
@@ -24,10 +24,10 @@
         "ext-soap": "*",
         "ext-curl": "*",
         "ass/xmlsecurity": "~1.0",
-        "symfony/framework-bundle": "^3.4|^4.0",
-        "symfony/twig-bundle": "^3.4|^4.0",
-        "zendframework/zend-mime": "2.1.*",
-        "symfony/dependency-injection": "^3.4|^4.0"
+        "symfony/dependency-injection": "^3.4 || ^4.0"
```
Comment (smithandre) [inline composer.json]: Comma missing, making the json invalid :/

**Review by smithandre — APPROVED** [review]

**Pull request merged**